src/fold/loop/common.py

```diff
@@ -204,29 +205,30 @@ def process_minibatch_transformation(
 
     if not is_X_available(X) and transformation.properties.requires_X:
         raise ValueError(
-            f"X is None, but transformation {transformation.__class__.__name__} requires it."
+            "X is None, but transformation"
+            f" {transformation.__class__.__name__} requires it."
         )
 
-    X_with_memory, y_with_memory = __preprocess_X_y_with_memory(transformation, X, y)
+    X_with_memory, y_with_memory = preprocess_X_y_with_memory(transformation, X, y)
     # The order is:
     # 1. fit (if we're in the initial_fit stage)
     if stage == Stage.inital_fit:
         transformation.fit(X_with_memory, y_with_memory, sample_weights)
-        __postprocess_X_y_into_memory(
+        postprocess_X_y_into_memory(
```

Comment:
moved the memory functions in a separate file. need to write unit test next time I'm touching them (there's a tweak how to speed up some of it)